try {
    coco
} catch (error) {
    console.log(error.message)
} finally {
    console.log("termino")
}
